.github/workflows/release-please.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+      - main
+name: release-please
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    permissions:
```

Comment:
This repository is currently [configured](https://github.com/eslint/generator-eslint/settings/actions) to give workflows write access by default. Specifying only the precise [`$GITHUB_TOKEN` permissions](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs) that release-please needs for this job reduces surface area and makes this workflow copy-pasteable to repositories that only grant read access by default.